Dockerfile

```diff
@@ -1,23 +1,28 @@
 FROM ubuntu:14.04
 
+ARG cdh_version
+ENV CDH_VERSION=$cdh_version
+
 # Prevent docker's default encoding of ASCII.
 # https://oncletom.io/2015/docker-encoding/
 ENV LANG C.UTF-8
 ENV LANGUAGE en_US:C
 ENV LC_ALL C.UTF-8
 
+# Install amazon's corretto packaging of openJDK
 RUN apt-get update && apt-get install -y python3-software-properties software-properties-common curl
-RUN add-apt-repository ppa:webupd8team/java
+RUN sudo apt-get install java-common
+RUN curl -s https://d3pxv6yz143wms.cloudfront.net/8.232.09.1/java-1.8.0-amazon-corretto-jdk_8.232.09-1_amd64.deb -o java-1.8.0-amazon-corretto-jdk_8.232.09-1_amd64.deb
+RUN sudo dpkg --install java-1.8.0-amazon-corretto-jdk_8.232.09-1_amd64.deb
```

Comment:
Just curious, why are we using Amazon's packaging for OpenJDK rather than the official OpenJDK? Although I couldn't find any issues in the official documentation for Corretto, but few things comes to mind:
- Cloudera's website mentions support for official OpenJDK 1.8 for CDH> 5.16. I am not aware of what things are changed in Corretto packaging (the documents do mention some changes done over official OpenJDK) but would we have to look out for some things not working as expected (specially configs related to JVM)?
- As we chose CDH packaging of Hadoop, it has somehow limited our ability to upgrade the different underlying softwares like HBase , Spark (we're stuck with the version CDH provides) and also OpenJDK. So, one more dependency on a packaged software provided by third party company makes me a bit skeptical.

Reply:
I just used that package because that's what @vmahedia had looked at / suggested.  I suspect he looked at that thinking that it might be used with EMR when we move to AWS?   It should be fine switching to the standard package as though; shouldn't be too difficult

Reply:
I modified the packaging from amazon to ppa

Reply:
Yea not sure why @vmahedia suggested that - but yea let's use the standard package.
Thank you for changing that.